src/main/java/com/udemy/elearning/configs/SecurityConfiguration.java

```diff
@@ -35,7 +35,8 @@ public class SecurityConfiguration {
             "api/courseReviews/**",
             "api/users/validate/**",
             "api/users/list",
-            "api/checkout/**"
+            "api/checkout/**",
```

Comment:
do not add "api/checkout/**", into whiltelist , we need to have bearer token

Reply:
OK bong.

Reply:
I'm already remove bong.